machine_configuration/README.md

```diff
@@ -1,51 +1,75 @@
+Due to a known incompatible issue with the Intel i915 driver, the below command is used to blacklist (disable loading of) the ast driver on worker nodes in the `intel-dgpu` MachineConfigPool. This is needed prior to loading Intel data center GPU drivers and firmware. 
 
-Becuase of the known incompatible issue from intel i915 driver, This command is used to blacklist (disable loading) of ast driver on worker nodes in Intel dGPU machine config pool. This is needed prior to loading Intel dGPU drivers and firmware. 
+Note: The driver that needs to be blacklisted will vary on the hardware platform the user is using. As a result, please use this [yaml file](/machine_configuration/100-intel-dgpu-machine-config-disable-ast.yaml) and below command as a reference for your environment.
 
-Note: This known incompatible issue will be fixed when upgrade to intel data center gpud driver - 1.0.0 using RHEL-9.x i915 driver
+Note: This command will reboot all the worker nodes in the `intel-dgpu` MachineConfigPool sequentially. This known incompatible issue will be addressed when Intel data center GPU drivers arrive for RHEL 9.x.
+
+```
+$ oc apply -f https://github.com/intel/intel-technology-enabling-for-openshift/blob/main/machine_configuration/100-intel-dgpu-machine-config-disable-ast.yaml
+```
+
+To verify the in-tree ast driver has been blacklisted on nodes with Intel data center GPU card, run `chroot /host` and then `lsmod | grep ast` on the node terminal via OCP web console (Compute -> Nodes -> Select a node that has the GPU card -> Terminal) or CLI. Ensure no output is returned by the command. If ast is still loaded, the command output will contain ast.
 
 ### Machine Configuration for Provisioning Intel QAT
 
-* Create QAT machine config pool:
+* Create `intel-qat` machine config pool:
+
+The command below creates a custom `intel-qat` MachineConfigPool for worker nodes with Intel QAT feature. These nodes are labeled `intel.feature.node.kubernetes.io/qat: 'true'` by NFD. 
+
+```
+$ oc apply -f https://github.com/intel/intel-technology-enabling-for-openshift/blob/main/machine_configuration/intel-qat-machine-config-pool.yaml
+```
 
-Following command creates a custom machine config pool for worker nodes with Intel QAT feature. These nodes are labeled `intel.feature.node.kubernetes.io/qat: 'true'` by NFD. 
+To verify the MachineConfigPool has been successfully created, navigate to the Compute -> MachineConfigPools section in the web console or run `oc get mcp` via CLI. Ensure `intel-qat` MachineConfigPool is present.
 
-```$ oc apply -f https://github.com/intel/intel-technology-enabling-for-openshift/blob/main/machine_configuration/intel-qat-machine-config-pool.yaml```
+* Turn on `intel_iommu` kernel parameter for QAT
 
-* Turn on intel_iommu kernel parameter for QAT
+For QAT, `intel_iommu` is a kernel parameter that needs to be turned on using the following command.
```

Comment:
should be called "kernel boot parameter"

Reply:
Thanks, done.